TestModels/SimpleTypes/SimpleBlob/runtimes/rust/src/operation/get_blob/_get_blob_input.rs

```diff
@@ -0,0 +1,64 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+#[allow(missing_docs)] // documentation missing in model
+#[non_exhaustive]
+#[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
+pub struct GetBlobInput {
+    #[allow(missing_docs)] // documentation missing in model
+    pub value: ::std::option::Option<::dafny_runtime::Sequence<::std::primitive::u8>>,
```

Comment:
Ah this should be the native idiomatic Rust type, not the Dafny type, since it's the struct that customers will work with.